BaseApi.js

```diff
@@ -10,16 +9,16 @@ const endpoints = {
   projects: baseUrl + '/app-projects/',
   modules: baseUrl + '/app-modules/',
   rate: baseUrl + '/contenttypes/$contentTypeId/objects/$objectPk/ratings/',
-  comments: baseUrl + '/contenttypes/$contentTypeId/objects/$objectPk/comments/',
+  comments: baseUrl + '/contenttypes/$contentTypeId/objects/$objectPk/comments/'
```

Comment:
@khamui Since we seem to keep adding/removing the trailing commas: https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8 or https://www.jetbrains.com/help/rider/Trailing_Commas_Style.html :nerd_face: Would you be ok with just leaving them in next time?